### PR TITLE
1312: Subnational laos schools overlays have no data

### PR DESCRIPTION
**Pull request by chris-bes**
### Changes:

- Migrate incorrect denominator

### Screenshots:
https://github.com/beyondessential/tupaia-backlog/issues/1312#issuecomment-695876807

### CODE REVIEW

File: packages/database/src/migrations/20200921010041-FixLaosSchoolsSchoolIndicatorsEiESubNationalLevelsMapOverlaysEntityAggregation-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  condition: {
+    operator: '=',
+    value: 'Yes',
+  },
+};
+const NEW_DENOMINATOR_CONFIG = {
+  condition: '*',
+};
+
+exports.up = function(db) {
```
Comment (chris-bes) [inline packages/database/src/migrations/20200921010041-FixLaosSchoolsSchoolIndicatorsEiESubNationalLevelsMapOverlaysEntityAggregation-modifies-data.js:44]: The current config in dev is:
https://github.com/beyondessential/tupaia-backlog/issues/1312#issuecomment-695875916

The config for for numerator was updated recently to fit the new code in `aggregateAnalytics.js`, but seems like the config for the denominator was missed. Also, there were some new overlays added to the sub national groups and was using the old config (basically a migration config). So this migration is for updating both the numerator/denominator config to match the new code in `aggregateAnalytics.js`

**Review by Alastair-L — APPROVED** [review]
LGTM!

**Comment by edmofro** [comment]
@chris-bes how many map overlays is this for? If it's not too many to do by hand, we now consider it better practice to follow the [admin panel process](https://docs.beyondessential.com.au/books/all-things-data/page/general-process-for-updating-dashboards-and-map-overlays-through-the-tupaia-admin-panel), which also means it will be released in production more quickly

**Comment by chris-bes** [comment]
> @chris-bes how many map overlays is this for? If it's not too many to do by hand, we now consider it better practice to follow the [admin panel process](https://docs.beyondessential.com.au/books/all-things-data/page/general-process-for-updating-dashboards-and-map-overlays-through-the-tupaia-admin-panel), which also means it will be released in production more quickly

There are 40 map overlays for this (20 for district level and 20 for province level). I estimate it'd take around half a day? And then after testing passed we also have to update in production as well

**Comment by edmofro** [comment]
Ok yep too many! Please target this PR with the same migration as a hotfix, it will be released today or tomorrow morning https://github.com/beyondessential/tupaia/pull/1365

**Comment by chris-bes** [comment]
Created another PR as hotfix: https://github.com/beyondessential/tupaia/pull/1367

**Pull request closed without merging**